CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 1.5.3
+
+* Improve performance for large numbers of args. Parsing 10k args fell from
+  250ms to 50ms; parsing 100k args fell from 15s to 100ms.
```

Comment:
update these numbers?

Reply:
Removed them :) ... it's a bit awkward to get the numbers, and I plan on following up with a second improvement, looks like we can get about another 2x by using string matching instead of regexps. If it works for you (and if you think the numbers are worth having) I'll add them back then? Thanks.

Reply:
sgtm - just don't want to have bad numbers if we list them :D